bep032tools/generator/tests/test_BEP032Generator.py

```diff
@@ -184,7 +184,36 @@ def test_generate_example_structure(self):
     def doCleanups(self):
         initialize_test_directory(clean=True)
 
+class Test_FolderGeneration(unittest.TestCase):
+
+    def setUp(self):
+        self.test_dir = Path(initialize_test_directory(clean=True))
+        self.test_dir = self.test_dir / 'generateTest'
```

Comment:
It's bad practice to overwrite variables immediately after setting them. I suggest also a renaming here
```suggestion
        test_dir = Path(initialize_test_directory(clean=True))
        self.test_dir = test_dir / 'generateTest'  # this folder will not exist yet
```